packages/lesswrong/lib/collections/digests/helpers.ts

```diff
@@ -1,5 +1,12 @@
 import moment from "moment"
-import { combineUrls, getSiteUrl } from "../../vulcan-lib"
+import { capitalize, combineUrls, getSiteUrl } from "../../vulcan-lib"
+import { SettingsOption } from "../posts/dropdownOptions"
+import { TupleSet, UnionOf } from "../../utils/typeGuardUtils"
+import { DIGEST_STATUSES } from "../digestPosts/schema"
+import type { DigestPost } from "../../../components/ea-forum/digest/EditDigest"
+
+export const DIGEST_STATUS_OPTIONS = new TupleSet([...DIGEST_STATUSES, 'pending'] as const)
```

Comment:
I'm not sure if the word "pending" is clear - very open to other suggestions. (I started with "undecided" but I thought that was too interchangeable with "maybe".)